BioArchLinux/spades/PKGBUILD

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Maintainer: PumpkinCheshire <me at pumpkincheshire dot top>
+# Maintainer: alienzj <alienchuj at gmail dot com>
+
+pkgname=spades
+pkgver=3.15.5
+pkgrel=1
+pkgdesc='SPAdes Genome Assembler'
+url='http://cab.spbu.ru/software/spades/'
+arch=('x86_64')
+license=('GPLv2')
+depends=('python')
+makedepends=(
+    'gcc'
+    'cmake'
+    'make'
+    'pkgconf'
+    'zlib'
+    'bzip2'
+    'bash'
+    'openmp'
+)
+checkdepends=('python')
+provides=('spades')
```

Comment:
We don't need to write `provides` and `conflicts`. pkg `spades` provides `spades` by default. `spades-git` or `spades-bin`  would need to add `provides=(spades)` and `conflicts=(spades)`.

Reply:
Ok, updated. Thanks!